From feb97f65e201040c0f2f03014a248c2e93c12dff Mon Sep 17 00:00:00 2001
From: Marcin Wojtas <mw@semihalf.com>
Date: Fri, 2 Dec 2016 16:52:51 +0100
Subject: [PATCH 0644/1345] fix: arm64: dts: cp110: set telephony node to use
 ICU interrupts

commit  f70cff9a1609b4ecf37a208a4a0eeaef4846cbda from
https://github.com/MarvellEmbeddedProcessors/linux-marvell.git

This patch aligns TDM interrupt mapping to the latest ICU driver
which was added recently (allowing TDM interrupt to be functional).

Change-Id: I84db0c0ab74bd5ce38ef63d913f2930332f1fdbb
Signed-off-by: Marcin Wojtas <mw@semihalf.com>
Signed-off-by: Igal Liberman <igall@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/34634
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
Tested-by: Star_Automation <star@marvell.com>
Reviewed-by: Omri Itach <omrii@marvell.com>
Reviewed-by: Hanna Hawa <hannah@marvell.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/marvell/armada-cp110-1.dtsi |    6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/arch/arm64/boot/dts/marvell/armada-cp110-1.dtsi b/arch/arm64/boot/dts/marvell/armada-cp110-1.dtsi
index 5307719..8c58c67 100644
--- a/arch/arm64/boot/dts/marvell/armada-cp110-1.dtsi
+++ b/arch/arm64/boot/dts/marvell/armada-cp110-1.dtsi
@@ -153,9 +153,9 @@ tdm@7a0000 {
 	compatible = "marvell,armada-a8k-tdm";
 	reg = <0x7a0000 0x20000>;
 	reg-names = "tdm_regs";
-	interrupts = <GIC_SPI 80 IRQ_TYPE_LEVEL_HIGH>,
-		     <GIC_SPI 81 IRQ_TYPE_LEVEL_HIGH>,
-		     <GIC_SPI 82 IRQ_TYPE_LEVEL_HIGH>;
+	interrupts = <ICU_GRP_NSR 111 IRQ_TYPE_LEVEL_HIGH>,
+		     <ICU_GRP_NSR 112 IRQ_TYPE_LEVEL_HIGH>,
+		     <ICU_GRP_NSR 113 IRQ_TYPE_LEVEL_HIGH>;
 	clocks = <&cps_syscon0 1 1>;
 	clock-names = "gateclk";
 	status = "disabled";
-- 
1.7.9.5

